src/RestControllerTrait.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Yiisoft\Yii\Rest;
+
+use Psr\Http\Message\ResponseInterface;
+
+trait RestControllerTrait
```

Comment:
Is it useful for non-tests?

Reply:
hmm...may be yes when IDE will suggest you already defined method signature, eg. you typed `ge` and IDE suggests you
 ```php
public function get(): ResponseInterface
{
    // ...
}
```